data/profiles/default.yaml

```diff
@@ -233,6 +233,54 @@ autoInjection:
           rewriteAppHTTPProbe: false
           selfSigned: false
 
+# Istio Gateway feature
+gateways:
+  enabled: true
+  components:
+    ingressGateway:
+    - gateway:
+        common:
```

Comment:
Do the default gateway ports need to be added here? 

* https://github.com/istio/installer/blob/67b0877a322bca62cbd1ebd9e48a5f748b534507/gateways/istio-ingress/values.yaml#L22-L50
* https://github.com/istio/installer/blob/67b0877a322bca62cbd1ebd9e48a5f748b534507/gateways/istio-ingress/values.yaml#L98-L108
* https://github.com/istio/installer/blob/67b0877a322bca62cbd1ebd9e48a5f748b534507/gateways/istio-ingress/values.yaml#L123-L147

Also, should this include other advanced options like `ISTIO_META_ROUTER_MODE: "sni-dnat"`, zvpn?

Reply:
@ayj thanks, I have raised another PR: https://github.com/istio/operator/pull/76 to migrate other profiles(demo, demo-auth, minimal...) to operator and have applied your comments into that PR.
And in this PR, I will only add feature and components for gateways.